modules/dapps.js

```diff
@@ -286,7 +286,7 @@ shared.getGenesis = function(req, cb, tx) {
 		{
 			id: req.dappid,
 		},
-		{},
+		{ sort: 'rowId:asc', limit: 1 },
```

Comment:
I don't think this sorting param will work.
The name of the field is `t_rowId` and I don't think we should use `t_rowId` here as well.

I've just noticed that transactions entity is already using `t_rowId:asc` as default.
The sort param can be removed.

Reply:
Done. Resolving. @lsilvs please check PR again.